### PR TITLE
Add --image-tags flag to support custom tags in Docker images

### PR DESCRIPTION
**Pull request by adibrastegarnia**
To address issue #42

### CODE REVIEW

**Comment by adibrastegarnia** [comment]
> There's some duplicate code in here. I think we should be extending the `imageName` function which already adds the registry prefix.

@kuujo 
I think I addressed your comment. Please take a look and let me know if you need more changes.

**Comment by kuujo** [comment]
hmm... maybe we need flags for specific images? Maybe it makes more sense to do `--controller-image`, `--raft-image`, `--config-image`, `--topo-image`, `--device-simulator-image`, etc

File: pkg/onit/cli/create.go [hunk 1]
```diff
@@ -60,6 +85,9 @@ func getCreateClusterCommand() *cobra.Command {
 			partitions, _ := cmd.Flags().GetInt("partitions")
 			partitionSize, _ := cmd.Flags().GetInt("partition-size")
 			configName, _ := cmd.Flags().GetString("config")
+			ImageTags, _ := cmd.Flags().GetStringToString("image-tags")
```
Comment (kuujo) [inline pkg/onit/cli/create.go]: I think it's convention to use `lowerCamel` for local variables? Or is that just in my head?

Comment (adibrastegarnia) [reply pkg/onit/cli/create.go]: Yes. that is correct. Will be fixed.

**Pull request merged**